OpenVpnService.sln

```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012

```

Comment:
It looks like MonoDevelop (which I use) and Visual Studio (used by someone else) are fighting each other here. I can revert this change if necessary.